### PR TITLE
feat: main page navigation cards, nav panel v2, team section v2

### PR DESCRIPTION
**Pull request by dudendy**
Resolves: #463

### CODE REVIEW

**Review by skv2rock — APPROVED** [review]

File: src/data/routes.data.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    ExplorerCardIcon,
+    FakeGalleryCardIcon,
+    GlossaryCardIcon,
+    GuildsCardIcon,
+    LendCardIcon,
+    MapCardIcon,
+    MarketCardIcon,
+    RafflesCardIcon
+} from 'components/Icons/Icons';
+import { Route } from 'shared/models';
```
Comment (dmutrokot) [inline src/data/routes.data.tsx:14]: `shared` should go first

File: src/data/routes.data.tsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+export const routes: Route[] = [
```
Comment (dmutrokot) [inline src/data/routes.data.tsx:16]: `routes` - will be better to name `sideNavRoutes` or something like this. The same with model `Route`

File: src/pages/Main/components/NavigationCards/NavigationCards.tsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
+import { NavLink } from 'react-router-dom';
+
+import { routes } from 'data/routes.data';
```
Comment (dmutrokot) [inline src/pages/Main/components/NavigationCards/NavigationCards.tsx:3]: please sort and group imports

File: src/root/NavPanel/NavPanel.tsx [hunk 4]
```diff
@@ -1,118 +1,33 @@
...
-import TravelExploreIcon from '@mui/icons-material/TravelExplore';
-
-import classNames from 'classnames';
 
 import { useAppSelector } from 'core/store/hooks';
 import { getActiveAddress } from 'core/store/login';
-import { GuildIcon, LendingIcon, CraftIcon, GlossaryIcon, FakeGotchisIcon } from 'components/Icons/Icons';
+
+import { routes } from 'data/routes.data';
+import { Route } from 'shared/models';
```
Comment (dmutrokot) [inline src/root/NavPanel/NavPanel.tsx:7]: should be placed before `data` import

**Review by dmutrokot — CHANGES_REQUESTED** [review]

**Review by dmutrokot — APPROVED** [review]

**Pull request merged**